be/src/exec/pipeline/exchange/split_local_exchange.h

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+#include <queue>
+
+#include "column/vectorized_fwd.h"
+#include "exec/pipeline/source_operator.h"
+#include "exprs/expr_context.h"
+#include "multi_cast_local_exchange.h"
+namespace starrocks::pipeline {
+class SplitLocalExchangeSinkOperator;
+// ===== exchanger =====
```

Comment:
better to add some comments to explain its usage scenarios and basic design ideas